### ISSUE TITLE
Change dashboard link to logo

### ISSUE THREAD

**Issue by amgalanbadmaev** [freeats/freeats#251]
**Problem**
We are lacking brand identity with our UI.

**Solution**
Mockups: [Figma](https://www.figma.com/design/kQOH4s5DnfWL0p6o6tlMQz/Draft-%5BATS%5D?node-id=1988-18689&t=UaJl7jYZypOuyxBw-1)

- Change "Dashboard" text link to our square logo.
- Should be done similar to [Tabler](https://preview.tabler.io/navigation.html) (whole container should be 48px height; no left-padding, right-padding is 16px).
- On mobile: do not display logo, display as regular "Dashboard" link text in menu; search input should be kept

Logo file (keep original resolution):

![logo_stack](https://github.com/user-attachments/assets/7c1072aa-23b7-4ece-ab65-ddee4e6dc03b)

**Comment by amgalanbadmaev** [comment]
Logo focus border doesn't have the same style as other nav-item links:

![CleanShot 28 11 2024 at 18-27](https://github.com/user-attachments/assets/f1f4f338-e576-4602-9bd0-578e4b67d147)

![CleanShot 28 11 2024 at 18-28](https://github.com/user-attachments/assets/fa0bb0be-622e-46d0-aeac-b91e5293c326)

Will this be solved in separate issue? I.e. this one - https://github.com/freeats/freeats/issues/258

**Comment by PrinceYauj** [comment]
> Logo focus border doesn't have the same style as other nav-item links:
> Will this be solved in separate issue? I.e. this one - #258

Yes, I've added the point `Also check a focus ring around the logo` in your issue
